config/gateway/conf.d/proxy_config/list.conf

```diff
@@ -0,0 +1,4 @@
+location ~ /list {
+    rewrite /list(.*) /api/v1$1 break;
+    proxy_pass http://list-service-server:5000;
```

Comment:
@Edison-Stuart Up to you, but you may consider dropping the `server` suffix from the hostnames. There's only one API for each, so denoting the `server` as opposed to another API associated with the service is not required (and it keeps the names short). Matter of style, up to your preference.